getStartsAndEnds.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Created by dusting on 11/4/15.
+ */
+var _ = require('underscore');
+
+var internal = exports.internal = {};
```

Comment:
O(n^2) which I know is the worst. I suppose I could find another way to tag them.